mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,20 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = (req, res) => { 
+  Order.findByIdAndDelete(req.params.orderId) 
+    .then(deletedOrder => { 
+      if (!deletedOrder) {
+        return res.status(404).json({
+          error: "Order not found", 
+        });
+      }
+      res.json({ message: "Order deleted successfully" }); 
+    })
+    .catch(error => { 
```

Comment:
Error handling is not consistent throughout the codebase. In some places, `errorHandler` is used to handle errors, but in this catch block, the error is not handled in the same way. It should be handled consistently throughout the codebase. Consider using `errorHandler` here as well to maintain consistency.